buildSrc/src/main/kotlin/com/virtuslab/gitmachete/buildsrc/IntellijPlugin.kt

```diff
@@ -43,7 +58,7 @@ fun Project.configureIntellijPlugin() {
     pluginDescription.set(file("$rootDir/DESCRIPTION.html").readText())
 
     changeNotes.set(
-      "<h3>v${rootProject.version}</h3>\n\n${file("$rootDir/CHANGE-NOTES.html").readText()}"
+      "<h3>v${rootProject.version}</h3>\n\n${changelog.getUnreleased().toHTML()}"
```

Comment:
Hmmm should `Unreleased` be used here? Shouldn't it rather be:
```
if unreleased non empty: unreleased
else: latest version
```
Coz now on `master` builds, the `Unreleased` section is supposed to be **empty**, right?